atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -145,7 +145,8 @@ public void run() {
                         log.error("The background sweep job failed unexpectedly with a batch size of "
                                 + ((int) (batchSizeMultiplier * sweepRowBatchSize.get()))
                                 + ". Attempting to continue with a lower batch size...", e);
-                        batchSizeMultiplier = Math.min(batchSizeMultiplier / 2, 1.0f / sweepRowBatchSize.get());
+                        // Cut batch size in half, always sweep at least one row (we round down).
+                        batchSizeMultiplier = Math.max(batchSizeMultiplier / 2, 1.5f / sweepRowBatchSize.get());
```

Comment:
Why move from 1.0f to 1.5f?  Doesn't seem it would make much of a difference?  Is that hope that it would be safer in that it would help avoid underflow?